Template/Tag/MatomoTag.web.js

```diff
@@ -148,6 +148,8 @@
                     if (matomoConfig.trackVisibleContentImpressions) {
                         tracker.trackVisibleContentImpressions();
                     }
+                    
+                    tracker.setSiteId(matomoConfig.idSite);
```

Comment:
maybe it should be rather just below the call to `setCookiePath` as the functions above (alwaysUseSendBeacon, enableLinkTracking, etc.) may need the id site to function properly.

Reply:
var applyFirst = ['addTracker', 'disableCookies', 'setTrackerUrl', 'setAPIUrl', 'enableCrossDomainLinking', 'setCrossDomainLinkingTimeout', 'setSecureCookie', 'setCookiePath', 'setCookieDomain', 'setDomains', 'setUserId', 'setSiteId', 'alwaysUseSendBeacon', 'enableLinkTracking', 'requireConsent', 'setConsentGiven'];
this is the order